lib/rubocop/cop/rspec/output.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # NOTE: This is the same as the `Rails/Output` cop with minor changes.
```

Comment:
Could you please delete this comment? It will diverge significantly from the actual Rails/Output cop in the future.